scripts/set_vars.sh

```diff
@@ -12,7 +12,19 @@ export REPO_PATH="${REPO_PATH:-${PLATFORM_PATH}/repo.path}"
 export SC_MACHINE_REPO="${SC_MACHINE_REPO:-https://github.com/ostis-ai/sc-machine.git}"
 export SC_MACHINE_BRANCH="${SC_MACHINE_BRANCH:-main}"
 export SC_MACHINE_PATH="${SC_MACHINE_PATH:-${PLATFORM_PATH}/sc-machine}"
+export SC_MACHINE_NAME="${SC_MACHINE_NAME:-sc-machine}"
 
 export SC_WEB_REPO="${SC_WEB_REPO:-https://github.com/ostis-ai/sc-web.git}"
 export SC_WEB_BRANCH="${SC_WEB_BRANCH:-main}"
 export SC_WEB_PATH="${SC_WEB_PATH:-${PLATFORM_PATH}/sc-web}"
+export SC_WEB_NAME="${SC_WEB_NAME:-sc-web}"
+
+if [ -d "${SC_MACHINE_PATH}" ];
```

Comment:
if I got your logic correctly, those `if` statements will never execute because `SC_MACHINE_PATH` and `SC_WEB_PATH` always have a default value

Reply:
This vars must be setted before their scripts will be executed